### PR TITLE
feat: REPLAT-11397 drop cap starting with quote for web

### PR DESCRIPTION
**Pull request by infreezer**
Capitalising the first two characters if the first one is:
image

Before:
image

After:
image

### CODE REVIEW

**Comment by infreezer** [comment]
Please advise if the idea to detect the first letter or digit and capitalize everything before it is okay.

File: packages/article-skeleton/src/dropcap-util-common.js [hunk 1]
```diff
@@ -12,6 +12,13 @@ const splitNode = node => {
     return node;
   }
   if (children[0].name === "text") {
+    const firstLetterOrDigit = children[0].attributes.value.match(
+      "[a-zA-Z0-9]"
+    );
+    const firstLetterOrDigitIndex = firstLetterOrDigit
```
Comment (apostolnikov) [inline packages/article-skeleton/src/dropcap-util-common.js]: Are we sure that the drop cap-ed word can only start with letter or a digit?

File: packages/article-skeleton/src/dropcap-util-common.js [hunk 2]
```diff
@@ -22,15 +29,20 @@ const splitNode = node => {
           ...children[0],
           attributes: {
             ...children[0].attributes,
-            value: children[0].attributes.value.slice(0, 1),
+            value: children[0].attributes.value.slice(
+              0,
+              firstLetterOrDigitIndex + 1
+            ),
```
Comment (apostolnikov) [inline packages/article-skeleton/src/dropcap-util-common.js]: We might want to compute the plus one adding where the constant is created, so we can avoid doing the operation twice.

**Comment by apostolnikov** [comment]
> Please advise if the idea to detect the first letter or digit and capitalize everything before it is okay.

I think the approach is good, the only thing that we need to check is whether or not we can have some symbol instead of digit or letter.

File: packages/article-skeleton/src/dropcap-util-common.js (hunk 1 above)
Comment (infreezer) [reply packages/article-skeleton/src/dropcap-util-common.js]: Let's check this with Kumaran.

File: packages/article-skeleton/src/dropcap-util-common.js (hunk 2 above)
Comment (infreezer) [reply packages/article-skeleton/src/dropcap-util-common.js]: Please verify.

**Pull request merged**